### ISSUE TITLE
2025-03-24, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#119]
# Today's Arxiv Papers

(1) [Dancing with Critiques: Enhancing LLM Reasoning with Stepwise Natural Language Self-Critique](http://dx.doi.org/10.13140/RG.2.2.27912.33289), 2025-03-21

(2) [OpenVLThinker: An Early Exploration to Complex Vision-Language Reasoning via Iterative Self-Improvement](http://arxiv.org/abs/2503.17352v1), 2025-03-21

(3) [Capturing Individual Human Preferences with Reward Features](http://arxiv.org/abs/2503.17338v1), 2025-03-21

(4) [Efficient Intent-Based Filtering for Multi-Party Conversations Using Knowledge Distillation from LLMs](http://arxiv.org/abs/2503.17336v1), 2025-03-21

(5) [CVE-Bench: A Benchmark for AI Agents' Ability to Exploit Real-World Web Application Vulnerabilities](http://arxiv.org/abs/2503.17332v1), 2025-03-21

(6) [LLM+MAP: Bimanual Robot Task Planning using Large Language Models and Planning Domain Definition Language](http://arxiv.org/abs/2503.17309v1), 2025-03-21

(7) [Bugdar: AI-Augmented Secure Code Review for GitHub Pull Requests](http://arxiv.org/abs/2503.17302v1), 2025-03-21

(8) [CASE -- Condition-Aware Sentence Embeddings for Conditional Semantic Textual Similarity Measurement](http://arxiv.org/abs/2503.17279v1), 2025-03-21

(9) [SafeMERGE: Preserving Safety Alignment in Fine-Tuned Large Language Models via Selective Layer-Wise Model Merging](http://arxiv.org/abs/2503.17239v1), 2025-03-21

(10) [FactSelfCheck: Fact-Level Black-Box Hallucination Detection for LLMs](http://arxiv.org/abs/2503.17229v1), 2025-03-21

(11) [Automating Adjudication of Cardiovascular Events Using Large Language Models](http://arxiv.org/abs/2503.17222v1), 2025-03-21

(12) [TreeSynth: Synthesizing Diverse Data from Scratch via Tree-Guided Subspace Partitioning](http://arxiv.org/abs/2503.17195v1), 2025-03-21

(13) [LLMs Love Python: A Study of LLMs' Bias for Programming Languages and Libraries](http://arxiv.org/abs/2503.17181v1), 2025-03-21

(14) [Modifying Large Language Model Post-Training for Diverse Creative Writing](http://arxiv.org/abs/2503.17126v1), 2025-03-21

(15) [Large Language Model Compression via the Nested Activation-Aware Decomposition](http://arxiv.org/abs/2503.17101v1), 2025-03-21

(16) [Deterministic AI Agent Personality Expression through Standard Psychological Diagnostics](http://dx.doi.org/10.70235/allora.0x20015), 2025-03-21

(17) [A Study into Investigating Temporal Robustness of LLMs](http://arxiv.org/abs/2503.17073v1), 2025-03-21

(18) [PVChat: Personalized Video Chat with One-Shot Learning](http://arxiv.org/abs/2503.17069v1), 2025-03-21

(19) [Problem Framing in the AI era: a new model](http://arxiv.org/abs/2503.17040v1), 2025-03-21

(20) [Summarization Metrics for Spanish and Basque: Do Automatic Scores and LLM-Judges Correlate with Humans?](http://arxiv.org/abs/2503.17039v1), 2025-03-21

(21) [Text2Model: Generating dynamic chemical reactor models using large language models (LLMs)](http://arxiv.org/abs/2503.17004v1), 2025-03-21

(22) [A Survey on Personalized Alignment -- The Missing Piece for Large Language Models in Real-World Applications](http://arxiv.org/abs/2503.17003v1), 2025-03-21

(23) [Token Dynamics: Towards Efficient and Dynamic Video Token Representation for Video Large Language Models](http://arxiv.org/abs/2503.16980v1), 2025-03-21

(24) [Assessing Consistency and Reproducibility in the Outputs of Large Language Models: Evidence Across Diverse Finance and Accounting Tasks](http://arxiv.org/abs/2503.16974v1), 2025-03-21

(25) [When Words Outperform Vision: VLMs Can Self-Improve Via Text-Only Training For Human-Centered Decision Making](http://arxiv.org/abs/2503.16965v1), 2025-03-21

(26) [TEMPO: Temporal Preference Optimization of Video LLMs via Difficulty Scheduling and Pre-SFT Alignment](http://arxiv.org/abs/2503.16929v1), 2025-03-21

(27) [RustEvo^2: An Evolving Benchmark for API Evolution in LLM-based Rust Code Generation](http://arxiv.org/abs/2503.16922v1), 2025-03-21

(28) [FAIT: Fault-Aware Fine-Tuning for Better Code Generation](http://arxiv.org/abs/2503.16913v1), 2025-03-21

(29) [Improving the End-to-End Efficiency of Offline Inference for Multi-LLM Applications Based on Sampling and Simulation](http://arxiv.org/abs/2503.16893v1), 2025-03-21

(30) [Assessing the Reliability and Validity of GPT-4 in Annotating Emotion Appraisal Ratings](http://arxiv.org/abs/2503.16883v1), 2025-03-21

(31) [Federated Cross-Domain Click-Through Rate Prediction With Large Language Model Augmentation](http://arxiv.org/abs/2503.16875v1), 2025-03-21

(32) [MARS: A Multi-Agent Framework Incorporating Socratic Guidance for Automated Prompt Optimization](http://arxiv.org/abs/2503.16874v1), 2025-03-21

(33) [Lie Detector: Unified Backdoor Detection via Cross-Examination Framework](http://arxiv.org/abs/2503.16872v1), 2025-03-21

(34) [Sparse Logit Sampling: Accelerating Knowledge Distillation in LLMs](http://arxiv.org/abs/2503.16870v1), 2025-03-21

(35) [ETVA: Evaluation of Text-to-Video Alignment via Fine-grained Question Generation and Answering](http://arxiv.org/abs/2503.16867v1), 2025-03-21

(36) [MTBench: A Multimodal Time Series Benchmark for Temporal Reasoning and Question Answering](http://arxiv.org/abs/2503.16858v1), 2025-03-21

(37) [Towards LLM Guardrails via Sparse Representation Steering](http://arxiv.org/abs/2503.16851v1), 2025-03-21

(38) [LoRASculpt: Sculpting LoRA for Harmonizing General and Specialized Knowledge in Multimodal Large Language Models](http://arxiv.org/abs/2503.16843v1), 2025-03-21

(39) [When Tom Eats Kimchi: Evaluating Cultural Bias of Multimodal Large Language Models in Cultural Mixture Contexts](http://arxiv.org/abs/2503.16826v1), 2025-03-21

(40) [When Debate Fails: Bias Reinforcement in Large Language Models](http://arxiv.org/abs/2503.16814v1), 2025-03-21

(41) [Conversational User-AI Intervention: A Study on Prompt Rewriting for Improved LLM Response Generation](http://arxiv.org/abs/2503.16789v1), 2025-03-21

(42) [A-IDE : Agent-Integrated Denoising Experts](http://arxiv.org/abs/2503.16780v1), 2025-03-21

(43) [Chain-of-Tools: Utilizing Massive Unseen Tools in the CoT Reasoning of Frozen Language Models](http://arxiv.org/abs/2503.16779v1), 2025-03-21

(44) [Current and Future Use of Large Language Models for Knowledge Work](http://arxiv.org/abs/2503.16774v1), 2025-03-21

(45) [On Explaining (Large) Language Models For Code Using Global Code-Based Explanations](http://arxiv.org/abs/2503.16771v1), 2025-03-21

(46) [SuperARC: A Test for General and Super Intelligence Based on First Principles of Recursion Theory and Algorithmic Probability](http://arxiv.org/abs/2503.16743v1), 2025-03-20

(47) [Automated Harmfulness Testing for Code Large Language Models](http://arxiv.org/abs/2503.16740v1), 2025-03-20

(48) [Since U Been Gone: Augmenting Context-Aware Transcriptions for Re-engaging in Immersive VR Meetings](http://arxiv.org/abs/2503.16739v1), 2025-03-20

(49) [Towards Agentic Recommender Systems in the Era of Multimodal Large Language Models](http://arxiv.org/abs/2503.16734v1), 2025-03-20

(50) [Design and Implementation of an FPGA-Based Tiled Matrix Multiplication Accelerator for Transformer Self-Attention on the Xilinx KV260 SoM](http://arxiv.org/abs/2503.16731v1), 2025-03-20

(51) [Natural Language Generation](http://arxiv.org/abs/2503.16728v1), 2025-03-20

(52) [APPA : Agentic Preformulation Pathway Assistant](http://arxiv.org/abs/2503.16698v1), 2025-03-20

(53) [Echoes of Power: Investigating Geopolitical Bias in US and China Large Language Models](http://arxiv.org/abs/2503.16679v1), 2025-03-20

(54) [Through the LLM Looking Glass: A Socratic Self-Assessment of Donkeys, Elephants, and Markets](http://arxiv.org/abs/2503.16674v1), 2025-03-20

(55) [Accelerating Transformer Inference and Training with 2:4 Activation Sparsity](http://arxiv.org/abs/2503.16672v1), 2025-03-20

(56) [Code Evolution Graphs: Understanding Large Language Model Driven Design of Algorithms](http://arxiv.org/abs/2503.16668v1), 2025-03-20

(57) [Accelerating Antibiotic Discovery with Large Language Models and Knowledge Graphs](http://arxiv.org/abs/2503.16655v1), 2025-03-20

(58) [Leveraging Large Language Models for Explainable Activity Recognition in Smart Homes: A Critical Evaluation](http://arxiv.org/abs/2503.16622v1), 2025-03-20

(59) [Stop Overthinking: A Survey on Efficient Reasoning for Large Language Models](http://arxiv.org/abs/2503.16419v1), 2025-03-20

(60) [Survey on Evaluation of LLM-based Agents](http://arxiv.org/abs/2503.16416v1), 2025-03-20

(61) [M3: 3D-Spatial MultiModal Memory](http://arxiv.org/abs/2503.16413v1), 2025-03-20

(62) [The Emperor's New Clothes in Benchmarking? A Rigorous Examination of Mitigation Strategies for LLM Benchmark Data Contamination](http://arxiv.org/abs/2503.16402v1), 2025-03-20

(63) [Exploring the Hidden Reasoning Process of Large Language Models by Misleading Them](http://arxiv.org/abs/2503.16401v1), 2025-03-20

(64) [Deconstructing Long Chain-of-Thought: A Structured Reasoning Optimization Framework for Long CoT Distillation](http://arxiv.org/abs/2503.16385v1), 2025-03-20

(65) [LaPIG: Cross-Modal Generation of Paired Thermal and Visible Facial Images](http://arxiv.org/abs/2503.16376v1), 2025-03-20

(66) [CaKE: Circuit-aware Editing Enables Generalizable Knowledge Learners](http://arxiv.org/abs/2503.16356v1), 2025-03-20

(67) [LLM Braces: Straightening Out LLM Predictions with Relevant Sub-Updates](http://arxiv.org/abs/2503.16334v1), 2025-03-20

(68) [OmniGeo: Towards a Multimodal Large Language Models for Geospatial Artificial Intelligence](http://arxiv.org/abs/2503.16326v1), 2025-03-20

(69) [Issue2Test: Generating Reproducing Test Cases from Issue Reports](http://arxiv.org/abs/2503.16320v1), 2025-03-20

(70) [Bridging Technology and Humanities: Evaluating the Impact of Large Language Models on Social Sciences Research with DeepSeek-R1](http://arxiv.org/abs/2503.16304v2), 2025-03-20

(71) [Uni-3DAR: Unified 3D Generation and Understanding via Autoregression on Compressed Spatial Tokens](http://arxiv.org/abs/2503.16278v2), 2025-03-20

(72) [Chain of Functions: A Programmatic Pipeline for Fine-Grained Chart Reasoning Data](http://arxiv.org/abs/2503.16260v1), 2025-03-20

(73) [Plug-and-Play 1.x-Bit KV Cache Quantization for Video Large Language Models](http://arxiv.org/abs/2503.16257v1), 2025-03-20

(74) [Fin-R1: A Large Language Model for Financial Reasoning through Reinforcement Learning](http://arxiv.org/abs/2503.16252v2), 2025-03-20

(75) [Distributed LLMs and Multimodal Large Language Models: A Survey on Advances, Challenges, and Future Directions](http://arxiv.org/abs/2503.16585v1), 2025-03-20

(76) [Reinforcement Learning for Reasoning in Small LLMs: What Works and What Doesn't](http://arxiv.org/abs/2503.16219v1), 2025-03-20

(77) [MathFusion: Enhancing Mathematic Problem-solving of LLM through Instruction Fusion](http://arxiv.org/abs/2503.16212v1), 2025-03-20

(78) [Affective Polarization Amongst Swedish Politicians](http://arxiv.org/abs/2503.16193v2), 2025-03-20

(79) [Large Language Models for Water Distribution Systems Modeling and Decision-Making](http://arxiv.org/abs/2503.16191v1), 2025-03-20

(80) [CLS-RL: Image Classification with Rule-Based Reinforcement Learning](http://arxiv.org/abs/2503.16188v1), 2025-03-20

(81) [CodeReviewQA: The Code Review Comprehension Assessment for Large Language Models](http://arxiv.org/abs/2503.16167v1), 2025-03-20

(82) [SpeCache: Speculative Key-Value Caching for Efficient Generation of LLMs](http://arxiv.org/abs/2503.16163v1), 2025-03-20

(83) [Towards Lighter and Robust Evaluation for Retrieval Augmented Generation](http://arxiv.org/abs/2503.16161v1), 2025-03-20

(84) [Automatically Generating Chinese Homophone Words to Probe Machine Translation Estimation Systems](http://arxiv.org/abs/2503.16158v1), 2025-03-20

(85) [Only a Little to the Left: A Theory-grounded Measure of Political Bias in Large Language Models](http://arxiv.org/abs/2503.16148v1), 2025-03-20

(86) [Unify and Triumph: Polyglot, Diverse, and Self-Consistent Generation of Unit Tests with LLMs](http://arxiv.org/abs/2503.16144v1), 2025-03-20

(87) [Investigating Retrieval-Augmented Generation in Quranic Studies: A Study of 13 Open-Source Large Language Models](http://dx.doi.org/10.14569/IJACSA.2025.01602134), 2025-03-20

(88) [MKG-Rank: Enhancing Large Language Models with Knowledge Graph for Multilingual Medical Question Answering](http://arxiv.org/abs/2503.16131v2), 2025-03-20

(89) [The Impact of Revealing Large Language Model Stochasticity on Trust, Reliability, and Anthropomorphization](http://arxiv.org/abs/2503.16114v1), 2025-03-20

(90) [Cultural Alignment in Large Language Models Using Soft Prompt Tuning](http://arxiv.org/abs/2503.16094v1), 2025-03-20

(91) [Tuning LLMs by RAG Principles: Towards LLM-native Memory](http://arxiv.org/abs/2503.16071v1), 2025-03-20

(92) [Meta-Learning Neural Mechanisms rather than Bayesian Priors](http://arxiv.org/abs/2503.16048v1), 2025-03-20

(93) [Incomplete Utterance Rewriting with Editing Operation Guidance and Utterance Augmentation](http://arxiv.org/abs/2503.16043v1), 2025-03-20

(94) [GreenIQ: A Deep Search Platform for Comprehensive Carbon Market Analysis and Automated Report Generation](http://arxiv.org/abs/2503.16041v2), 2025-03-20

(95) [Evaluating Test-Time Scaling LLMs for Legal Reasoning: OpenAI o1, DeepSeek-R1, and Beyond](http://arxiv.org/abs/2503.16040v1), 2025-03-20

(96) [Hybrid-Level Instruction Injection for Video Token Compression in Multi-modal Large Language Models](http://arxiv.org/abs/2503.16036v1), 2025-03-20

(97) [The Lighthouse of Language: Enhancing LLM Agents via Critique-Guided Improvement](http://arxiv.org/abs/2503.16024v1), 2025-03-20

(98) [BadToken: Token-level Backdoor Attacks to Multi-modal Large Language Models](http://arxiv.org/abs/2503.16023v1), 2025-03-20

(99) [Corrective In-Context Learning: Evaluating Self-Correction in Large Language Models](http://arxiv.org/abs/2503.16022v1), 2025-03-20

(100) [Autonomous AI imitators increase diversity in homogeneous information ecosystems](http://arxiv.org/abs/2503.16021v2), 2025-03-20
